src/App.vue

```diff
@@ -4,5 +4,142 @@
   </div>
 </template>
 
-<style>
+<style lang="scss">
+@import "assets/scss/variables";
+
+// Google Fonts.
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:400,400i,700,700i');
+@import url('https://fonts.googleapis.com/css?family=Montserrat:400,700');
+
+// Base Styles.
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-family: $opensans;
+}
+
+h1, h2, h3 {
+  color: $background3;
+  font-family: $montserrat;
+  letter-spacing: 1px;
+}
+
+h1 {
+  font-size: 3rem;
+}
+
+h2 {
+  font-size: 2.4rem;
+}
+
+h3 {
+  font-size: 2rem;
+}
+
+p {
+  font-family: $opensans;
+  font-size: 1.4rem;
+  letter-spacing: 1px;
+  color: $text;
+}
+
+strong {
+  color: $text;
+  font-weight: 700;
+}
+
+a {
+  font-family: $opensans;
+  font-size: 1.4rem;
+  font-weight: 700;
+  color: #2DBCDB;
+}
+
+.button {
+  display: inline-block;
+  text-align: center;
+  background: none;
+  border: none;
+  cursor: pointer;
+  font-weight: 700;
+  font-family: $montserrat;
+  box-sizing: border-box;
+  text-decoration: none;
+  color: #fff;
+  font-size: 2rem;
+}
+
+.button-xlarge {
+  width: 263px;
```

Comment:
Avoid fixed dimensions, it doesn't accomodate long texts or multiple lines of text. Use paddings, but allow buttons to grow freely as needed.